include/flag.h

```diff
@@ -48,7 +48,7 @@ struct flag {
     boolean showexp;         /* show experience points */
     boolean showscore;       /* show score */
     boolean silent;          /* whether the bell rings or not */
-    boolean sortloot;        /* sort items alphabetically when looting */
```

Comment:
This was more a change to make Clang happy, as there are a few places where `sortloot` is treated more like a `char` that a boolean type (I use a custom prefix header that typedefs `boolean` to C99's `bool`).

Will be reverting anyway.